### ISSUE TITLE
Mod5 infra-node show wrong EC2 instance type (M4.2xlarge vs M5.2xlarge)

### ISSUE THREAD

**Issue by netzzer** [openshift/openshift-cns-testdrive#490]
Mod5, MachineSets, Machines, and Nodes, for command below shows m4.2xlarge in labguide. Current RHPDS workshop has M5.2xlarge EC2 instances. 

```
$ oc get machine -n openshift-machine-api
NAME                                              STATE     TYPE         REGION      ZONE
         AGE
cluster-ocs4-f002-dwmf9-master-0                  running   m4.xlarge    us-east-2   us-e
ast-2a   118m
cluster-ocs4-f002-dwmf9-master-1                  running   m4.xlarge    us-east-2   us-e
ast-2b   118m
cluster-ocs4-f002-dwmf9-master-2                  running   m4.xlarge    us-east-2   us-e
ast-2c   118m
cluster-ocs4-f002-dwmf9-worker-us-east-2a-md4k5   running   m5.2xlarge   us-east-2   us-e
ast-2a   117m
cluster-ocs4-f002-dwmf9-worker-us-east-2b-zhd97   running   m5.2xlarge   us-east-2   us-e
ast-2b   117m
cluster-ocs4-f002-dwmf9-worker-us-east-2c-bw4f4   running   m5.2xlarge   us-east-2   us-e
ast-2c   117m
```

**Comment by netzzer** [comment]
Scaling machineset has same issue, labguide shows m4.2xlarge.

```
cluster-ocs4-f002-dwmf9-worker-us-east-2c-l2q5g   pending   m5.2xlarge   us-east-2   us-e
ast-2c   8s
```

**Comment by thoraxe** [comment]
will be fixed by #499
